circle.yml

```diff
@@ -76,6 +76,7 @@ test:
         - rake ci:run[win32_event_log]
         - rake ci:run[kafka_consumer]
         - rake ci:run[kafka]
+        - rake ci:run[docker_daemon]
```

Comment:
we do need it, they're actually not mocked, we bring up a couple of containers during these tests.